spec/tools/miqldap_to_sssd/miqldap_configuration_spec.rb

```diff
@@ -0,0 +1,28 @@
+$LOAD_PATH << Rails.root.join("tools", "miqldap_to_sssd").to_s
+
+require "miqldap_configuration"
```

Comment:
Is this needed if you are modifying the `$LOAD_PATH` above?

Reply:
Seems to be, yes. Without it the following error is produced: `uninitialized constant MiqLdapToSssd`